src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,783 @@
+    factoryModels.add(new Infantry());
+    factoryModels.add(new Tank());
+    factoryModels.add(new Missiles());
+    factoryModels.add(new Rockets());
+    factoryModels.add(new AA());
+    factoryModels.add(new Radar());
+    factoryModels.add(new Artillery());
+    factoryModels.add(new Maser());
+    factoryModels.add(new LigerPanther());
+    factoryModels.add(new SuperZ2());
+    extras.add(new SuperZ2Hurt());
+    factoryModels.add(new OGRPlatform());
+
+    // Record those units we can get from a Seaport.
+    kaijuModels.add(new KaijuWarsKaiju.Alphazaurus());
+    final KaijuWarsKaiju.HellTurkey     turkeyAir  = new KaijuWarsKaiju.HellTurkey();
+    final KaijuWarsKaiju.HellTurkeyLand turkeyLand = new KaijuWarsKaiju.HellTurkeyLand(turkeyAir);
+    final KaijuWarsKaiju.HellTurkeyEgg  turkeyEgg  = new KaijuWarsKaiju.HellTurkeyEgg(turkeyLand);
+    turkeyAir.turkeyLand = turkeyLand;
+    turkeyAir.turkeyEgg = turkeyEgg;
+    kaijuModels.add(turkeyAir);
+    extras.add(turkeyLand);
+    extras.add(turkeyEgg);
+    final KaijuWarsKaiju.BigDonkRampage rampager = new KaijuWarsKaiju.BigDonkRampage();
+    extras.add(rampager);
+    kaijuModels.add(new KaijuWarsKaiju.BigDonk(rampager));
+    final KaijuWarsKaiju.SnekTunneler tunneler = new KaijuWarsKaiju.SnekTunneler();
+    extras.add(tunneler);
+    kaijuModels.add(new KaijuWarsKaiju.Snek(tunneler));
+    final KaijuWarsKaiju.UFOAbducts abductor = new KaijuWarsKaiju.UFOAbducts();
+    extras.add(abductor);
+    kaijuModels.add(new KaijuWarsKaiju.UFO(abductor));
+
+    // Inscribe those war machines obtainable from an Airport.
+    airportModels.add(new Fighter());
+    airportModels.add(new Bomber());
+    airportModels.add(new Helicopter());
+    airportModels.add(new Bushplane());
+
+    // Carrier gets to build all normal air units
+    UnitModel carrier = new SkyCarrier();
+    for (UnitModel um : airportModels)
+      carrier.baseActions.add(1, new UnitProduceLifecycle.UnitProduceFactory(um));
+
+    airportModels.add(new BigBoy());
+    airportModels.add(new GuncrossWing());
+    airportModels.add(new Kaputnik());
+    extras.add(new GuncrossRobot());
+    airportModels.add(carrier);
+
+    // Dump these lists into a hashmap for easy reference later.
+    kjwModels.shoppingList.put(TerrainType.FACTORY, factoryModels);
+    kjwModels.shoppingList.put(TerrainType.SEAPORT, kaijuModels);
+    kjwModels.shoppingList.put(TerrainType.AIRPORT, airportModels);
+
+    // Compile one master list of everything we can build.
+    for (UnitModel um : factoryModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : airportModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : kaijuModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : extras)
+      kjwModels.unitModels.add(um);
+
+    // Multiplies all movement (and ranges) by N
+    for( UnitModel um : kjwModels.unitModels )
+      um.baseMovePower *= KaijuWarsWeapons.KAIJU_SCALE_FACTOR;
```

Comment:
Why? Was this just for prototyping with different values before deciding that 1 is best and we don't need the variable after all? It is used as a multiplier for a number of seemingly-unrelated things.

Reply:
Yeah, it was just for prototyping.
I wasn't sure whether I liked 2 or 1 better, but it can probably go away now due to factors like "Kaiju exist now" and "Tactic bonuses make 2x less good"